.github/workflows/backup-db.yml

```diff
@@ -1,4 +1,4 @@
-name: Backup database to Azure storage
+name: Backup database to Azure storage with sanitised data
```

Comment:
can we change from backup-db.yml to backup-db-sanitised.yml? or similar. The non sanitised backup can stay as backup-db.yml (not backup-db-main.yml).